code/game/objects/items/weapons/sharpener.dm

```diff
@@ -36,7 +36,8 @@
 		user << "<span class='notice'>[I] has already been refined before. It cannot be sharpened further.</span>"
 		return
 	user.visible_message("<span class='notice'>[user] sharpens [I] with [src]!</span>", "<span class='notice'>You sharpen [I], making it much more deadly than before.</span>")
-	I.sharpness = IS_SHARP//this is only kept here because of super sharpening blocks, it wont do anything with standard since objects should already be sharp
+	if(!requires_sharpness)
```

Comment:
Bad. Don't forget to check for !I.sharpness. Otherwise the super version will dull knives.